bin/export_grafana_snapshot.py

```diff
@@ -0,0 +1,182 @@
+import argparse
+import datetime
+import json
+import os
+import time
+
+import requests
+import requests.auth
```

Comment:
pl Add a requirements file in the dir and add dependencies there.

Reply:
Ensure you do autopep8 on the file, and add it to ci/cd

Reply:
Added requirements file and did autopep8 check. Can you clarify how to add it to ci/cd